File: scratch/check_braces.js
const fs = require('fs');

function checkBraces(filepath) {
    if (!fs.existsSync(filepath)) {
        console.log(`File ${filepath} not found`);
        return;
    }
    
    const content = fs.readFileSync(filepath, 'utf8');
    const openBraces = (content.match(/\{/g) || []).length;
    const closeBraces = (content.match(/\}/g) || []).length;
    
    console.log(`File: ${filepath}`);
    console.log(`Open: ${openBraces}`);
    console.log(`Close: ${closeBraces}`);
    if (openBraces !== closeBraces) {
        console.log("!!! UNBALANCED BRACES !!!");
    }
}

checkBraces('public/styles.css');
checkBraces('public/index.html');
checkBraces('public/admin.html');

File: scratch/check_braces.py
import os

def check_braces(filepath):
    if not os.path.exists(filepath):
        print(f"File {filepath} not found")
        return
    
    with open(filepath, 'r', encoding='utf-8') as f:
        content = f.read()
        
    open_braces = content.count('{')
    close_braces = content.count('}')
    
    print(f"File: {filepath}")
    print(f"Open: {open_braces}")
    print(f"Close: {close_braces}")
    if open_braces != close_braces:
        print("!!! UNBALANCED BRACES !!!")

check_braces('public/styles.css')
check_braces('public/index.html')
check_braces('public/admin.html')
